modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc

```diff
@@ -0,0 +1,552 @@
+
+[width="65%",cols="~,~",options="header"]
+|===
+| Helm Chart Version
+| Works with Ininite Scale Versions
+
+| {tab_1_tab_text}
+| 2.0.0-beta.4 - 2.0.0-beta.5
+
+ifdef::use_tab_2[]
+| {tab_2_tab_text}
+| 2.0.0-beta.6 - 2.0.0-beta.7
+endif::[]
+
+ifdef::use_tab_3[]
+| {tab_3_tab_text}
+| 2.0.0-beta.8 - 2.0.0-beta.9
+endif::[]
+|===
+
+=== Get the Chart
+
+As the Helm chart has currently not been published to a Helm repository, you need to clone ownClouds Helm chart git repository named {ocis-helm-charts-url}[ocis-charts].  
+
+// note that the sources for the include are are on github.
+
+* Check the supported Kubernetes versions before you download the chart.
++
+Note the `~` represents all patch releases for that particular version:
++
+[tabs]
+====
+{tab_1_tab_text}::
++
+--
+include::{ocis-charts-raw-url}{tab_1}{kube-versions-url}[]
+--
+ifdef::use_tab_2[]
+{tab_2_tab_text}::
++
+--
+include::{ocis-charts-raw-url}{tab_2}{kube-versions-url}[]
+--
+endif::[]
+ifdef::use_tab_3[]
+{tab_3_tab_text}::
++
+--
+include::{ocis-charts-raw-url}{tab_3}{kube-versions-url}[]
+--
+endif::[]
+====
+
+=== Deploy the Chart
+
+// note that the sources for values and description are are on github.
+
+:t1_text: Helm chart with default configurations.
+:t2_text: Description for the values.yaml file.
+
+* Deploy the chart with the deployment name `ocis`, use any name as desired. To do so, run the following command from the root of the cloned repository:
++
+[source,bash]
+----
+helm install ocis ./charts/ocis
+----
++
+[tabs]
+====
+{tab_1_tab_text}::
++
+--
+[width="100%",cols="~,~",options="header"]
+|===
+| File
+| Descripton
+
+| xref:deployment/container/orchestration/values-tab-1.adoc[values.yaml,window=_blank]
+| {t1_text}
+
+| xref:deployment/container/orchestration/val-desc-tab-1.adoc[Values Description,window=_blank]
+| {t2_text}
+|===
+--
+ifdef::use_tab_2[]
+{tab_2_tab_text}::
++
+--
+[width="100%",cols="~,~",options="header"]
+|===
+| File
+| Descripton
+
+| xref:deployment/container/orchestration/values-tab-2.adoc[values.yaml,window=_blank]
+| {t1_text}
+
+| xref:deployment/container/orchestration/val-desc-tab-2.adoc[Values Description,window=_blank]
+| {t2_text}
+|===
+--
+endif::[]
+ifdef::use_tab_3[]
+{tab_3_tab_text}::
++
+--
+[width="100%",cols="~,~",options="header"]
+|===
+| File
+| Descripton
+
+| xref:deployment/container/orchestration/values-tab-3.adoc[values.yaml,window=_blank]
+| {t1_text}
+
+| xref:deployment/container/orchestration/val-desc-tab-3.adoc[Values Description,window=_blank]
+| {t2_text}
+|===
+--
+endif::[]
+====
+
+=== Start minikube
+
+. Start your minikube cluster:
++
+[source,bash]
+----
+minikube start
+----
+
+. Enable the minikube ingress plugin, which is like a reverse proxy for your cluster:
++
+[source,bash]
+----
+minikube addons enable ingress
+----
+
+=== Customise the Generic Setup
+
+In all examples, adapt the settings according your needs.
+
+==== Set Own Default Values
```

Comment:
```suggestion
==== Set Your Own Default Values
```

I am nor sure what to write here to make the heading grammatically good, but also it does not feel correct to have the word "Your" in a heading.